### DISCUSSION TITLE
Why jsx syntax?

### DISCUSSION THREAD

**Discussion by shivam-rcfs** [vercel/satori#188] [Q&A] [answered]
I started playing around and i got stuck when it comes to projects other then react/next. I was trying to use it in sveltekit project but jsx isn't supported in sveltekit/svelte. What kind of workaround we can have here??

**Comment by styfle** [comment, marked as answer]
See the [Use without JSX](https://github.com/vercel/satori/blob/main/README.md#use-without-jsx) section of the README.
 
You could also try [satori-html](https://github.com/natemoo-re/satori-html). However, that will parse html at runtime for each request. 

JSX avoids any html parsing at runtime because the JSX is transformed at build time.

**Reply by shivam-rcfs** [reply]
Thank you so much between please look into #185
